contrib/scripts/check-fmt.sh

```diff
@@ -3,13 +3,15 @@
 set -e
 set -o pipefail
 
+_goroot=$(${GO:-go} env GOROOT)
```

Comment:
This is pretty elegant.. hmm.

So far I got away without GOROOT, personally I find it's a bit opinionated and a pain to deal with.. maybe we could set the default for `_goroot` to "" iff `$GO` is not set..?

Reply:
FWIW, the Go toolchain will always return a non-empty value for "go env GOROOT", even if $GOROOT isn't set in the env. It'll derive it relative to the go binary invoked, unless it's overriden by some other means.

Reply:
:exploding_head: I totally thought I ran this and got errors out, but now that I try it today it's fine. -ENO:coffee: I guess!